project/common/project.go

```diff
@@ -0,0 +1,48 @@
+package common
+
+import (
+	"github.com/buildpacks/lifecycle/api"
+
+	"github.com/buildpacks/pack/internal/dist"
+)
+
+type Script struct {
+	API    string `toml:"api"`
```

Comment:
These are internal only representations right? Meaning these won't be `Marshalled()`, `Unmarshalled()` to the filesystem?

If so, then these `toml:"..."` annotations won't be used.

Reply:
I'm comfortable straddling both sides and treating `common` primarily as the internal representations with the (un)marshalling until there is a deviation between different versions.